### ISSUE TITLE
TanDEM-X filtering

### ISSUE THREAD

**Issue by dshean** [GLIMS-RGI/rgitools#37]
Noticed some artifacts in the RGI-TOPO figures of TanDEM-X 90-m DEM.  I recommend the using a series of filters based on values in the AUX products.  The workflow tested for HMA is in the `tandemx_mask.py` script here: https://github.com/dshean/tandemx.

**Comment by matthiasdusch** [comment]
Yes, the artefacts in the example figures are kept on purpose to show the restraints of the 'out-of-the-box'-TanDEMx.
Also for the final product, one DEM file per RGI glacier, I think we should keep the DEM as close to the original release as possible. That way people using them know what processing has happened and can also spot and deal with artefacts according to their needs.

But: It might be a good idea to either include some of the filtering techniques or at least link to repos like yours.

**Comment by fmaussion** [comment]
Thanks again! 

Yes, we should either use your tools or at the very least refer to them. As it is now, it's very unlikely that we will ever use TANDEM-X in production, but Bas mentioned this: https://github.com/GLIMS-RGI/rgitools/issues/35

**Comment by fmaussion** [comment]
Leaving this open as "wontfix", because it's very likely that we'll use COPDEM  over TANDEM-X for RGI7. If this changes we can reassess
